test/simplify_qdq_test.cpp

```diff
@@ -162,7 +165,7 @@ TEST_CASE(qdq_different_scales)
         auto t2     = m1.add_parameter("t2", sh2);
         auto scale1 = m1.add_literal(0.5f);
         auto scale2 = m1.add_literal(0.4f);
-        auto zero   = m1.add_literal(std::int8_t{0});
+        auto zero   = m1.add_literal(std::int8_t{1});
```

Comment:
This is no longer a variable called `zero`.  It is initialized to a `1`.  Maybe it should be called `zp`? Also, are you changing its value so it doesn't get eliminated?